### ISSUE TITLE
QA_EXPT-3 & EXPT-4_Further readings_content-is-same

### ISSUE THREAD

**Issue by BSravanthi** [Virtual-Labs/analog-cmos-vlsi-circuit-design-iiith#12]
Defect Description :
In the Further readings page of 3rs and 4th experiments of this lab, found that the content is same.

Actual Result :
In the Further readings page of 3rs and 4th experiments of this lab, found that the content is same.

Environment :
OS: Windows 7, Ubuntu-16.04,Centos-6
Browsers:Firefox-42.0,Chrome-47.0,chromium-45.0
Bandwidth : 100Mbps
Hardware Configuration:8GBRAM ,
Processor:i5

Attachments:
![qa_oe_avlsi_i11](https://cloud.githubusercontent.com/assets/13479177/26630230/72dc4bd6-4623-11e7-8cda-f2c8b2d9da91.png)

**Comment by sravanthimodepu** [comment]
Yes.. The 3rd & 4th experiments belongs to the same experiment. So the content is same, but the simulations are different. Found the same from the vlab.co.in page.

**Comment by BSravanthi** [comment]
Need to contact lab developer. The issue will be opened till we get the response.
